app/src/interfaces/_system/system-filter/input-group.vue

```diff
@@ -21,12 +23,28 @@ const fieldsStore = useFieldsStore();
 const relationsStore = useRelationsStore();
 const { t } = useI18n();
 
+const { info: collectionInfo } = useCollection(computed(() => props.collection));
+
+const versioningEnabled = computed(() => collectionInfo.value?.meta?.versioning ?? false);
+
+const { fakeVersionField } = useFakeVersionField(
```

Comment:
Use `toRef` instead of `computed` and set `includeChoices` parameter to `true`.

```suggestion
const { fakeVersionField } = useFakeVersionField(toRef(props, 'collection'), ref(true), versioningEnabled, true);
```